### DISCUSSION TITLE
安装不上？

### DISCUSSION THREAD

**Discussion by stoneken18** [tata1661/FSL-Mate#44] [Q&A]
很想用你们的paddlefsl模块，但安装后import paddlefsl总出错，ModuleNotFoundError: No module named 'paddlehelix'，又试着装paddlehelix,  按网上的教程根本装不上，请问你们能有什么建议吗？

**Comment by Ren-Xin** [comment]
@stoneken18 
export SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True
pip install rdkit-pypi
pip install pgl
pip install paddlehelix
